static/js/util.js

```diff
@@ -63,6 +63,9 @@ exports.same_stream_and_topic = function util_same_stream_and_topic(a, b) {
 };
 
 exports.is_current_user = function (email) {
+    if (email === null || email === undefined) {
```

Comment:
Thanks for the feedback. This change is tied to the modification for `bots_data__get_all()` in `bot_data.js`, wouldn't it be better for them to be in a single commit? Also I don't think the tests will pass if only one of these is in a commit.